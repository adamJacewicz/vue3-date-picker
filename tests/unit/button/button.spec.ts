import { shallowMount } from '@vue/test-utils';
import CButton    from '@/components/atoms/button.vue';

describe('c-button', () => {
  const button = shallowMount(CButton, {
    slots: {
      default: "Mock button!"
    }
  });

  it('render component with correct class', () => {
    expect(button.classes()).toContain('c-button');
  });

  it('render correct content', () => {
    expect(button.text()).toEqual('Mock button!');
  });

  it('should match snapshot', () => {
    expect(button.html()).toMatchSnapshot();
  });
});
