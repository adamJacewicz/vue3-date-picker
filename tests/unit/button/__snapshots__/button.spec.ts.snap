// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`c-button should match snapshot 1`] = `<button class="c-button">Mock button!</button>`;
